docs/management/admin/host-isolation-ov.asciidoc

```diff
@@ -41,14 +41,16 @@ To isolate a host from a case alert:
 
 To isolate a host from an endpoint:
 
-. Go to *Manage -> Endpoints*, then select the appropriate endpoint in the *Endpoint* column. The endpoint details flyout opens.
-. Click *Take action -> Isolate host*.
+. Go to *Manage -> Endpoints*, then either:
+    * Select the appropriate endpoint in the *Endpoint* column, then click *Take action -> Isolate host* in the endpoint details flyout.
```

Comment:
```suggestion
    * Select the appropriate endpoint in the *Endpoint* column, and click *Take action -> Isolate host* in the endpoint details flyout.
```
Petty but figured this might be a bit more readable since 'then' is used in the previous sentence. If you happen to agree there's another instance further down the same page.